src/content/pages/version-support.md

```diff
@@ -23,7 +23,8 @@ ESLint major release lines are designated by the level of support they receive f
 
 | **Release Line** | **Status** | **First Release** | **Last Release** | **EOL Start** | **Commercial Support** |
 |-------------------|------------|-------------------|-----------------|---------------|------------------------|
-| v9.x             | Current    | 2024-04-05        | TBD              | TBD | [Tidelift][tidelift] |
+| v10.x            | Current    | 2025-11-14<br>(prerelease)        | TBD              | TBD | [Tidelift][tidelift] |
```

Comment:
Since the First Release date should be the date of v10.0.0 final release, I added the `(prerelease)` note.